### ISSUE TITLE
Doesn't work in Gnome 42

### ISSUE THREAD

**Issue by IouSASTU** [Gniourf/Uptime-Indicator#12]
Upgraded to Ubuntu 22.04 LTS and so Gnome is now at 42, and now the extension will not load. 

Perhaps just needs to have the metadata.json updated?

**Comment by IouSASTU** [comment]
Yes after changing the metadata.json file to say 42 your extension is now running for me in Gnome 42 / Ubuntu 22.04 LTS.

**Comment by rxchit** [comment]
Show's this error in Extensions app
```Object 0x18cbb3225970 is not a subclass of GObject_Object, it's a Object```

Gnome 42.1, Manjaro testing,
added 42 in metadata.json

**Comment by Gniourf** [comment]
> Show's this error in Extensions app `Object 0x18cbb3225970 is not a subclass of GObject_Object, it's a Object`
> 
> Gnome 42.1, Manjaro testing, added 42 in metadata.json

I don't have Manjaro installed (I only have debian bullseye and ubuntu 22.04, and works well there), so it's hard to tell what went wrong. I just bumped the version to gnome 42, can you please try to install the extension from  the extensions website https://extensions.gnome.org/extension/508/uptime-indicator/ or the extension app `extension-manager` and see what happens there?
